std_msgs/Header header          # 头部信息
uint8           control_mode    # 运行模式
AUVPose         pose            # AUV位姿
AUVPose         target          # 目标位姿 
AUVTime         time            # AUV时间
AUVMotor        motor_force     # AUV电机力矩
AUVSensor       sensor          # AUV传感器数据
